pkg/command/command.go

```diff
@@ -0,0 +1,54 @@
+package command
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+)
+
+type Command struct {
+	SimpleCommands []SimpleCommand
+	// the following will make it easy to redirect using pipe and > or <
+	Output               io.Writer
+	Input                io.Reader
+	ErrOutput            io.Writer
+	CurrentCommand       *Command
+	CurrentSimpleCommand *SimpleCommand
+}
+
+type SimpleCommand struct {
```

Comment:
Command is the pkg name so remove it from the name? It becomes `command.Simple`.

Reply:
Thanks for noticing! I totally forgot that haha